azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -45,14 +65,13 @@ public ImageVersionDaoImpl(DatabaseOperator databaseOperator, ImageTypeDao image
 
   @Override
   public int createImageVersion(ImageVersion imageVersion) {
-    ImageType imageType = imageTypeDao.getImageTypeByType(imageVersion.getType())
+    ImageType imageType = imageTypeDao.getImageTypeByName(imageVersion.getName())
         .orElseThrow(() -> new RuntimeException("Unable to fetch image type metadata. Invalid "
-            + "image type : "+imageVersion.getType()));
+            + "image type : "+imageVersion.getName()));
     final SQLTransaction<Long> insertAndGetSpaceId = transOperator -> {
-      String currentTime = DateTime.now().toLocalDateTime().toString();
       transOperator.update(INSERT_IMAGE_VERSION_QUERY, imageVersion.getPath(), imageVersion.getDescription(),
           imageVersion.getVersion(), imageType.getId(), imageVersion.getState().getStateValue(),
-          imageVersion.getReleaseTag(), currentTime, imageVersion.getCreatedBy());
+          imageVersion.getReleaseTag(), imageVersion.getCreatedBy(), imageVersion.getCreatedBy());
```

Comment:
getCreatedBy is used two times. One should be changed to getModifiedBy

Reply:
As this is insert, both createdBy and modifiedBy will be same. That's why createdBy is passed. But to make it cleaner so that it is not confusing, let me update imageVersion with modifiedBy and change and pass modifiedBy here as well.